src/modules/imageresizer/ImageResizerContextMenu/Resources.resx

```diff
@@ -117,8 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="Resize_Pictures_Title" xml:space="preserve">
-    <value>Resize pictures</value>
+  <data name="ImageResizer_Context_Menu_Entry" xml:space="preserve">
+    <value>Resize with Image Resizer</value>
+    <comment>Image Resizer is the product name, do not loc</comment>
```

Comment:
so we are not localizing Image Resizer after all?

Reply:
You're right. Image Resizer is localized in Settings in plenty of places. Going to revert that comment.